application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -137,8 +141,15 @@ private static boolean isTitleValid(CharSequence title) {
     }
 
     private RestAction<User> createForumPost(ModalInteractionEvent event, User originalUser) {
-        MessageCreateData forumMessage = MessageCreateData
-            .fromContent(event.getValue(TRANSFER_QUESTION_INPUT_ID).getAsString());
+
+        String originalMessage = event.getValue(TRANSFER_QUESTION_INPUT_ID).getAsString();
+
+        String modifiedMessageForPost = String.format("Original Post from: %s %n %n %s",
+                originalUser.getAsMention(), originalMessage);
```

Comment:
(thats not what i suggested, reads weird like that - and why are there newlines?)

Reply:
please adjust the embed to look more natural. see for example this:

![example](https://i.imgur.com/OHqaoBA.png)

Reply:
so use the proper way of setting the embed author (with avatar and name). then put the message as message into it. u dont need any title or footer.

Reply:
> (thats not what i suggested, reads weird like that - and why are there newlines?)

wanted to seperate the question from other stuff so it's easier to read.

Reply:
thats fair, but there wont be any other stuff to separate from if you go with what i said. Essentially, you want the normal message content to be `@Foo has a question:` and then the embedd that should look like the original message without any extras.

The ping must be outside the embed, otherwise it does not ping. This is similar to `/tag ... reply:@John` which looks like this:
![demo](https://i.imgur.com/WuJzLSi.png)